src/components/Homepage/HomePage.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+
```

Comment:
Remove this unnecessary new lines. I highly recommend to use prettier extension to do all the code formatting

Reply:
noted.